tournamentcontrol/competition/tests/test_competition_site.py

```diff
@@ -353,3 +353,166 @@ def test_season_calendar_disabled(self):
             team.division.season.competition.slug,
             team.division.season.slug,
         )
+
+
+@override_settings(ROOT_URLCONF="tournamentcontrol.competition.tests.urls")
+class StreamInstructionsViewTests(TestCase):
+    """Test the stream-instructions view for both markdown and HTML formats."""
+
+    def setUp(self):
+        """Set up a superuser for permission-protected views."""
+        self.superuser = UserFactory.create(is_staff=True, is_superuser=True)
+
+    def test_stream_instructions_md_endpoint(self):
+        """Test that .md endpoint returns 200 and correct content-type."""
+        season = factories.SeasonFactory.create()
+
+        # Build the URL manually since the regex pattern doesn't reverse easily with format parameter
+        base_url = self.reverse(
+            "competition:season", season.competition.slug, season.slug
+        )
+        url = f"{base_url}stream-instructions.md"
```

Comment:
Never build URLs with string concatenation. Always use named URLs.

Also, make sure to pass the `name` and `args` to the `self.get` method, don't create `url` as an intermediate value.